packages/core/src/components/steps-modal/stories/steps-modal.core.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from '@storybook/web-components'
+import { html } from 'lit'
+
+import { ModalComponentArgs, ModalStoryArgs } from './steps-modal.args'
+
+export default {
+  title: 'Components/StepsModal',
+  ...ModalStoryArgs,
+} as Meta
+
+const createModal = () => {
+  return html`
+    <div>
+      <atom-button id="open-modal-steps">Open Modal</atom-button>
+      <atom-steps-modal
+        steps="3"
+        trigger="open-modal-steps"
+        steps-titles="Step 1, Step 2, Step 3"
+      >
+        <div slot="step-0">Step 1 Content</div>
+        <div slot="step-1">Step 2 Content</div>
+        <div slot="step-2">Step 3 Content</div>
+      </atom-steps-modal>
+    </div>
+  `
+}
+
+export const Default: StoryObj = {
```

Comment:
How about creating another one providing currentStep?

Reply:
done! [693539c](https://github.com/juntossomosmais/atomium/pull/599/commits/693539c1609736fc4c7eaffbc565d0631bc03425)

Reply:
question: when I click in button during last step is a good idea close the modal?

Reply:
Hmm, that's a good one. I guess not because we can use the last step to show any error and give the user the chance to define what to do next (close, try again). make sense?